logstash-core-event-java/build.gradle

```diff
@@ -1,18 +1,13 @@
-buildscript {
-    repositories {
-        mavenLocal()
-        mavenCentral()
-        jcenter()
-    }
-    dependencies {
-        classpath 'net.saliman:gradle-cobertura-plugin:2.2.8'
-    }
-}
+group = 'org.logstash'
+version = '6.0.0-alpha1'
```

Comment:
Can we get this version info from `versions.yml`? See https://github.com/elastic/logstash/pull/5882